tests/cpp/test_circular_buffering.cpp

```diff
@@ -712,20 +712,18 @@ TEST_P(CircularBufferingTest, SmemBlockGemmCache) {
   at::Tensor t1 = at::randn({K, N}, options);
   at::Tensor aten_output = at::matmul(t0.to(at::kDouble), t1.to(at::kDouble));
 
-  std::vector<c10::IValue> aten_inputs = {t0, t1};
-
   KernelExecutor ke;
-  ke.compile(&fusion, aten_inputs);
+  ke.compile(&fusion, {t0, t1});
```

Comment:
I'd prefer the old style, which would make it easier to change the arguments, whereas in the new code all of the compile, run and validate calls need to be maintained consistently. What is the motivation of this change?

Reply:
I'm about to change all the function signatures to be KernelArgumentHolder everywhere. I didn't want to send that PR changing them all simultaneously as it'd be hard to dig out the change of the API relative to the changes of the tests. This will temporarily make our tests more impervious to that type change when it comes.